src/libcore/python/main.cpp

```diff
@@ -73,25 +70,26 @@ PYBIND11_MODULE(core_ext, m) {
     if (!fr->contains(base_path))
         fr->append(base_path);
 
-    /* Holds a pointer to the current TBB task scheduler to enable
-       later changes via set_thread_count(). Cleaned up in
-       \ref cleanup_callback. */
-    std::unique_ptr<tbb::task_scheduler_init> *task_scheduler_holder =
-        new std::unique_ptr<tbb::task_scheduler_init>(
-            new tbb::task_scheduler_init());
-    m.def(
-        "set_thread_count",
```

Comment:
Would be nice to expose the `Thread::set_thread_count` function (or `pool_set_size` from enoki-thread) somehow. We may not always want to run the Python bindings with all threads

Reply:
It's already exposed by `enoki`:
https://github.com/wjakob/enoki/blob/5b5d5c9bbb45ab616054824217a1123d8f2d5e35/src/python/main.cpp#L207

Although we could rename that binding into `pool_set_size`.

Reply:
We could remove the `llvm_` prefix.